index.html

```diff
@@ -4,14 +4,29 @@
     <meta charset="utf-8">
     <title>Code Coop</title>
     <link rel="stylesheet" href="assets/css/index.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,700" rel="stylesheet">
   </head>
   <body>
     <header>
-        <h1>Code Coop</h1>
+        <!--
         <nav>
-            <a href="about.html" title="About page and what we do">About</a>
-            <a href="contact.html" title="Contact page with contact details">Contact Us</a>
+            <ul>
+                <li><a href="index.html" title="Home page">Home</a></li>
+                <li><a href="about.html" title="About page and what we do">About</a></li>
+                <li><a href="contact.html" title="Contact page with contact details">Contact Us</a></li>
+            </ul>
         </nav>
+        -->
+        <div class="main-header-container">
+          <h1 id="mainHeader">Code Co-Op</h1>
+        </div>
+        <hr>
+        <div class="catch-phrase-container">
+          <p id="catch-phrase">Lorem ipsum dolor sit amet consectetur, adipisicing elit.</p>
+        </div>
+        <div class="btn-container">
+          <button id="find-out-more-btn">Find out more</button>
```

Comment:
This was added as a 'what there could be' based of the pages we were using to model this off. Happy to change this if we think it will provide a different purpose in the future.